ansible/roles/edu-sharing/vars/versions/7.0.0.yml

```diff
@@ -0,0 +1,20 @@
+---
+edu_sharing_archive_url: https://artifacts.edu-sharing.com/repository/maven-remote/org/edu_sharing/edu_sharing-projects-community-deploy-docker-compose/7.0.0/edu_sharing-projects-community-deploy-docker-compose-7.0.0-bin.zip
+
+# Enable/Disable migration from previous Edu-Sharing version
+# This must run once and then please make it false
+edu_sharing_enable_migration: true
+#  This will create backups for old versions, since previous backup are not compatible
+# Enable/disable backup backup for old versions
+edu_sharing_enable_backup: true
+#  Backup  files 
+edu_sharing_sql_backup_file: '{{backup_path}}/{{alfresco_db.name}}-dump.sql.gz'
+edu_sharing_alfresco_backup_file: '{{ backup_path }}/alfresco-backup.tar.gz'
+# In newer versions Edu-sharing will change the deploy command so we need to update it here
+# deploy docker images to server
+edu_sharing_deploy_command: 'sg docker -c "./deploy.sh start"'
+# remove docker images from server
+edu_sharing_undeploy_command: sg docker -c "./deploy.sh stop"
+edu_sharing_purge_all_command: |
+     yes | sg docker -c "./deploy.sh remove"
+     yes | sg docker -c "./deploy.sh purge"
```

Comment:
Do you expect that the command will change on different instances? Or why do they need to be configurable? Couldn't we just use the commands as is directly in the taks?